src/DebugToFile.cpp

```diff
@@ -42,6 +42,8 @@ class DebugToFile : public IRMutator {
                 num_elements *= bound.extent;
             }
 
+            // TODO: why do we bother with this? halide_debug_to_file()
```

Comment:
This code predates halide_buffer_t, and buffer_t only had an elem_size, not a type code. Feel free to fix.

Reply:
> This code predates halide_buffer_t, and buffer_t only had an elem_size, not a type code. Feel free to fix.

I'll do that as a followup.